src/App.js

```diff
@@ -1,15 +1,42 @@
 import React from "react";
-
+import Top from "./Top";
+import Heading from "./Heading";
 import Bookings from "./Bookings";
+import InfoCard from "./InfoCard";
 import "./App.css";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
 
 const App = () => {
+  const contactDetails = [
```

Comment:
Can we move the contactDetails inside the Footer component? I don't think there is a need to define contactDetails here and pass it to the Footer component as props